examples/diabetes_example/run_diabetes.py

```diff
@@ -0,0 +1,510 @@
+   .. only:: builder_html or readthedocs
+
+        :download:`assets required to run this example <../../../../tmp/diabetes_assets.zip>`
+
+  Please ensure to have all the libraries installed. A *requirements.txt* file is included in the zip file, where you can run the command ``pip install -r requirements.txt`` to install them.
+
+"""
+
+# %%
+# Importing all the dependencies
+# ==============================
+
+import os
+import zipfile
+import pathlib
+
+import substra
+from substra.sdk.schemas import (
+    FunctionSpec,
+    FunctionInputSpec,
+    FunctionOutputSpec,
+    AssetKind,
+    DataSampleSpec,
+    DatasetSpec,
+    Permissions,
+    TaskSpec,
+    ComputeTaskOutputSpec,
+    InputRef,
+)
+
+# sphinx_gallery_thumbnail_path = 'static/example_thumbnail/diabetes.png'
+
+from assets.dataset.diabetes_dataset import setup_diabetes
+
+# %%
+# Instantiating the Substra clients
+# =================================
+#
+# We work with three different organizations.
+# Two organizations provide data, and a third one performs Federate Analytics to compute aggregated statistics without
+# having access to the raw datasets.
+#
+# This example runs in local mode, simulating a federated learning experiment.
+#
+
+
+# Choose the subprocess mode to locally simulate the FL process
+N_CLIENTS = 3
+clients = [substra.Client(backend_type="subprocess") for _ in range(N_CLIENTS)]
+clients = {client.organization_info().organization_id: client for client in clients}
+
+# Store organization IDs
+ORGS_ID = list(clients)
+
+# The provider of the functions for computing analytics is defined as the first organization.
+ANALYTICS_PROVIDER_ORG_ID = ORGS_ID[0]
+# Data providers orgs are the two last organizations.
+DATA_PROVIDER_ORGS_ID = ORGS_ID[1:]
+
+# %%
+# Creating and registering the assets
+# -----------------------------------
+#
+# Every asset will be created in respect to predefined schemas (Spec) previously imported from
+# ``substra.sdk.schemas``. To register assets, :ref:`documentation/api_reference:Schemas`
+# are first instantiated and the specs are then registered, which generate the real assets.
+#
+# Permissions are defined when registering assets. In a nutshell:
+#
+# - Data cannot be seen once it's registered on the platform.
+# - Metadata are visible by all the users of a network.
+# - Permissions allow you to execute a function on a certain dataset.
+#
+
+permissions = Permissions(public=False, authorized_ids=[ANALYTICS_PROVIDER_ORG_ID])
+# %%
+# Next, we need to define the asset directory. You should have already downloaded the assets folder as stated above.
+#
+# The function ``setup_diabetes`` downloads if needed the *diabetes* dataset, and split it in two. Each data organisation
+# has access to a chunk of the dataset.
+
+root_dir = pathlib.Path.cwd()
+assets_directory = root_dir / "assets"
+assert assets_directory.is_dir(), """Did not find the asset directory, a directory called 'assets' is
+expected in the same location as this file"""
```

Comment:
```suggestion
assert assets_directory.is_dir(), """Did not find the asset directory, a directory
called 'assets' is expected in the same location as this file"""
```

Reply:
break line earlier so that we don't have to horizontal scroll on RTD
![image](https://user-images.githubusercontent.com/5354301/223374182-e0c0ccdd-ab16-4254-90ac-c06aeecac35b.png)